Libraries/Image/RCTImageLoader.mm

```diff
@@ -54,20 +53,37 @@ static NSInteger RCTImageBytesForImage(UIImage *image)
 #endif // [TODO(macOS GH#774)
 }
 
-#if TARGET_OS_OSX
+#if TARGET_OS_OSX // [TODO(macOS GH#774)
+
+/**
+ * Github #1611 - We can't depend on RCTUIKit here, because this file's podspec (React-RCTImage) doesn't
+ * take a dependency on RCTUIKit's pod `React-Core`. Let's just copy the methods we want to shim here
+ */
```

Comment:
@mischreiber FYI, I didn't realize that many of our files can only depend on the header but not the implementation. That is nudging me slightly towards "new pod / clang module" over umbrella header. but as a 0.71 followup :)

Reply:
Feels like `React-RCTImage` _should_ depend on `React-Core`, unless there's already a reverse dependency (in which case these functions should probably move fully to `RCTImage`)? Also, could you clarify more what you mean by depending on header but not implementation?